plugins/luxtronic2/README.md

```diff
@@ -46,6 +56,12 @@ Defines a mapping to a attribute (read-only). All attribute values are bytes (nu
 ### lux2_c
 Defines a mapping to a calculated value (read-only). All calculated values are integer (numbers).
 
+### lux2_unpack
+Python lambda function. Called before the value is sent towards the device.
+
+### lux2_pack
+Called before the value is written to the smarthome item.
+
```

Comment:
This could also be a common setting for items - something like `eval_out` - which works like `eval`, but the other way around when item is set.